samples/contrib/pytorch-samples/Pipeline-Cifar10.ipynb

```diff
@@ -3,8 +3,6 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #185.        sleep_task = sleep_op(5).after(deploy_task).set_display_name("Sleep")
</pre><p>Will be good to add some comments on why the Sleep is needed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/pipelines/pull/6042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>